content/eksctl/launcheks.md

```diff
@@ -4,6 +4,25 @@ date: 2018-08-07T13:34:24-07:00
 weight: 20
 ---
 
+
+{{% notice warning %}}
+**DO NOT PROCEED** with this step unless you have [validated the IAM role](/prerequisites/workspaceiam/#validate-the-iam-role) in use by the Cloud9 IDE. You will not be able to run the necessary kubectl commands in the later modules unless the EKS cluster is built using the IAM role.
+{{% /notice %}}
+
+{{%expand "How do I check the IAM role on the workspace?" %}}
```

Comment:
You should use the Challenge template specified here: https://github.com/aws-samples/eks-workshop/wiki/Workshop-testing

Reply:
I'm concerned using a challenge question here. This is a critical step that should not be missed.